.github/workflows/pre-submit.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       run: make manifests bundle-k8s bundle-reset test
 
     - name: Test container build
```

Comment:
NIT: Can you add it here, so it would be easier to understand which container build are you referring to?
```suggestion
    - name: Test container build OCP Prometheus
```